client/src/api/UserAccessService/index.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * UserAccessService의 유즈케이스에서 사용할 fetch 함수를 정의합니다.
+ * 유즈케이스를 실제로 구현한 hook이 너무 커지는 것을 방지하기 위해 분리해두었습니다.
+ */
+
+import {
+  PostUserLoginRequestDto,
+  PostUserLoginResponseDto,
+  PostUserRegisterRequestDto,
+  PostUserRegisterResponseDto,
+} from "@core/dto/UserAccessService";
+
+export const postUserLogin = (
+  body: PostUserLoginRequestDto,
+): Promise<PostUserLoginResponseDto> =>
+  fetch(`/api/user/login`, {
+    method: "POST",
+    body: JSON.stringify(body),
+  }).then(async (res) => (res.ok ? res.json() : Promise.reject(res)));
```

Comment:
```suggestion
  }).then(async (res) => (res.ok ? res.json() : Promise.reject(res)));
```

생각해보니 await가 없는 코드에는 async를 사용하는 건 좋은 패턴은 아닌 것 같네요. 애초에 Promise에 then을 단 코드는 항상 Promise를 반환할테니 불필요할 것 같습니다

Reply:
아래도 전부 마찬가집니다

Reply:
`  }).then((res) => (res.ok ? res.json() : Promise.reject(res)));`
이렇게 바꾸면 된다는 거죠??

Reply:
아 넵넵 저거 잘못달았네요ㅋㅋㅋ